examples/list_objects.py

```diff
@@ -32,8 +32,8 @@
 
 # List all object paths in bucket that begin with my-prefixname using
 # V2 listing API.
-objects = client.list_objects_v2('my-bucketname', prefix='my-prefixname',
-                                 recursive=True)
+objects = client.list_objects('my-bucketname', prefix='my-prefixname',
```

Comment:
This example has duplicated code after your change, I guess you need to use api v1 in the first line and update some wording

Reply:
done